Knoema.Client/Client.cs

```diff
@@ -125,6 +125,11 @@ public Task<Dataset> GetDataset(string id)
 			return ApiGet<Dataset>(string.Format("/api/1.0/meta/dataset/{0}", id));
 		}
 
+		public Task<IEnumerable<Dataset>> GetDatasetsBySource(string source)
```

Comment:
This method is redundant. ListDatasets should be enhanced with optional parameters for source, topic and region filter.